reflex-dom-ace.cabal

```diff
@@ -21,16 +21,19 @@ library
     , containers
     , data-default
     , ghcjs-dom
+    , jsaddle
+    , jsaddle-dom
+    , lens
     , mtl
     , reflex
     , reflex-dom
     , text
 
   if impl(ghcjs)
-    build-depends: ghcjs-base
+    --build-depends: ghcjs-base
+    cpp-options: -D_GHCJS_ -DGHCJS_GC_INTERVAL=60000
```

Comment:
I vaguely remember that there is already a `__GHCJS__` flag defined in ghcjs case. There's also `ghcjs_HOST_OS`. If so, can we drop `_GHCJS_`?